catalog/test/disease_controller_tests.js

```diff
@@ -0,0 +1,263 @@
+const expect = require('chai').expect;
+const sinon = require('sinon');
+
+const Disease = require('../models/disease');
+const diseaseController = require('../controllers/disease');
+const testUtil = require('./test_util');
+const imagesUtil = require('../util/clear-images');
+
+describe('==== Disease Controller ====', function() {
+  describe('getDiseases()', function() {
+    let status, json, res;
+    beforeEach(function() {
+      status = sinon.stub();
+      json = sinon.spy();
+      res = { json, status };
+      status.returns(res);
+      sinon.stub(Disease, 'find');
+    });
+
+    afterEach(function() {
+      sinon.restore();
+    });
+
+    it('test_return_200_on_success', async function() {
+      await diseaseController.getDiseases({}, res, () => {});
+      expect(status.calledOnce).to.be.true;
+      expect(status.args[0][0]).to.equal(200);
+    });
+
+    it('test_body_contains_message_and_disease_objects', async function() {
+      Disease.find.returns(['Diseases data']);
+      await diseaseController.getDiseases({}, res, () => {});
+      expect(json.calledOnce).to.be.true;
+      expect(json.args[0][0].message).to.equal('Diseases fetched.');
+      expect(json.args[0][0].diseases).to.equal(Disease.find());
+    });
+
+    it('test_no_diseases_in_database', async function() {
+      Disease.find.returns([]);
+      await diseaseController.getDiseases({}, res, () => {});
+      expect(json.calledOnce).to.be.true;
+      expect(json.args[0][0].message).to.equal('Diseases fetched.');
```

Comment:
You don't need to test this. This is a duplicate since you've already checked for it in the teste case right above.